edgelet/edgelet-docker/test/linux/bad_settings.manual.cs.3.yaml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Remove this empty first line. Other files below also have it.

Reply:
All done.